.github/workflows/conformance-ingress.yaml

```diff
@@ -21,7 +21,6 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  minikube_version: 1.25.2
```

Comment:
No more Minikube version requirement?

Reply:
this version was originally added, but then I noticed that minikube is shipped by default in github runner. There is a risk of version drift, hope that my PR in minikube action will get merged soon.

https://github.com/medyagh/setup-minikube/pull/32